packages/config/config/devices/0x0371/zwa012.json

```diff
@@ -47,52 +63,142 @@
 			"$import": "~/0x0086/templates/aeotec_template.json#sensor_operation_mode"
 		},
 		"2": {
-			"$import": "~/0x0086/templates/aeotec_template.json#invert_state_report"
+			"$import": "~/0x0086/templates/aeotec_template.json#external_switch_3way",
```

Comment:
There's not really a point in doing that import when you're changing everything but the `minValue`. Just write this parameter normally if there is no good template for it.

Reply:
OK, done